GRAMMAR.md

```diff
@@ -92,10 +94,16 @@ filter.
 
 ```
      field
-     	: STANDARD_FIELD
-     	| CUSTOM_FIELD
-     	| function
-     	;
+       : field ADD field 
+       | field SUB field 
+       | field MUL field 
+       | field DIV field 
+       | field MOD field 
+       | STANDARD_FIELD
+       | CUSTOM_FIELD
+       | function
+       | numeric
+       ;
```

Comment:
A tricky part with these is that the field side  and the condition side are becoming more and more tide together. Previously the only place literals were allowed was within the functions, now we will be requiring it within arithmetic as well.